github-repo-manager/src/main/java/com/axonivy/github/file/GitHubMissingFiles.java

```diff
@@ -0,0 +1,27 @@
+package com.axonivy.github.file;
+
+import static com.axonivy.github.file.GitHubFiles.CODE_OF_CONDUCT;
+import static com.axonivy.github.file.GitHubFiles.LICENSE;
+import static com.axonivy.github.file.GitHubFiles.SECURITY;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.axonivy.github.file.GitHubFiles.FileMeta;
+
+public class GitHubMissingFiles {
+
+  private static final List<String> WORKING_ORGANIZATIONS = List.of("axonivy", "axonivy-market");
+  private static final List<FileMeta> REQUIRED_FILES = List.of(LICENSE, SECURITY, CODE_OF_CONDUCT);
+
+  public static void main(String[] args) throws IOException {
+    int missingStatus = 0;
+    for (var fileMeta : REQUIRED_FILES) {
+      var githubMissingFiles = new GitHubMissingFilesDetector(fileMeta, WORKING_ORGANIZATIONS);
+      var returnedStatus = githubMissingFiles.checkMissingFile();
+      missingStatus = returnedStatus != 0 ? returnedStatus : 0;
```

Comment:
If returnedStatus is not 0 in the first item and 0 in the second the result is 0 which is wrong. Code should be:
missingStatus = returnedStatus != 0? returnedStatus : missingStatus;

Reply:
ah ah sorry Reto, my bad.
I updated it, I have the same idea but I described it wrong.